tests/pre_analysis/test_plate_recording.py

```diff
@@ -346,11 +344,6 @@ def test_PlateRecording__loads_from_zipped_xlsx_correctly():
     assert all(pr.wells)
 
 
-def test_PlateRecording__raises_error_when_duplicate_wells_found_in_optical_files():
```

Comment:
Why remove this test? We should still be checking for duplicate wells right?

Reply:
Removed it from where it was being checked, can add a check somewhere else